reporting/impl/src/main/resources-mtr/reports/templates/include/about.ftl

```diff
@@ -0,0 +1,29 @@
+<dl class="dl-horizontal">
+    <#if reportModel.relatedResources.windupAbout.windupRuntimeVersion?has_content>
+        <dt>Generated By</dt>
+        <dd>${getWindupBrandName()} ${reportModel.relatedResources.windupAbout.windupRuntimeVersion}</dd>
+    </#if>
+
+    <dt>Website</dt>
+    <dd><a href="https://developers.redhat.com/products/mta/overview/">https://developers.redhat.com/products/mta/overview/</a></dd>
```

Comment:
We should create a ticket pointing to these lines of code to change the links when there's an MTR website available, since MTR pointing to the "old" MTA is not strictly correct I think

Reply:
Ticket created here https://issues.redhat.com/browse/WINDUP-3461 we can update the links once those are available